test.js

```diff
@@ -19,6 +19,11 @@ it("works", function()
 	expect( parseMetaRefresh("")                          ).to.deep.equal(nothing);
 	expect( parseMetaRefresh(" ")                         ).to.deep.equal(nothing);
 	expect( parseMetaRefresh("5")                         ).to.deep.equal(timeoutOnly);
+
+	expect( parseMetaRefresh(" 5")                         ).to.deep.equal(timeoutOnly);
+	expect( parseMetaRefresh("5 ")                         ).to.deep.equal(timeoutOnly);
+	expect( parseMetaRefresh(" 5 ")                        ).to.deep.equal(timeoutOnly);
```

Comment:
Now that mismatching encapsulators are accepted (which I've tested to work in Chrome, Firefox and Safari), we should add a test for it:
```js
expect( parseMetaRefresh(`5; 'http://domain.com/"`) ).to.deep.equal(timeoutAndUrl1);
```

Reply:
Can you provide a link to this new feature? Google hasn't been useful here. Is it just a mix of single and double quotes or are other chars supported?

Reply:
I was never able to find documentation on this. I merely tested everything in numerous browsers.

Reply:
Mixed encapsulation is now supported w/ dcf3b010726466ea57a241a00db441e8fb831352